raffles-bulletin-of-zoology.csl

```diff
@@ -0,0 +1,661 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>Raffles Bulletin of Zoology</title>
+    <title-short>RBZ</title-short>
+    <id>http://www.zotero.org/styles/raffles-bulletin-of-zoology</id>
+    <link href="http://www.zotero.org/styles/raffles-bulletin-of-zoology" rel="self"/>
+    <link href="http://lkcnhm.nus.edu.sg/nus/index.php/instructions" rel="documentation"/>
+    <author>
+      <name>Kin Guan Wee</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="zoology"/>
+    <category field="generic-base"/>
+    <updated>2017-09-06T00:55:28+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editortranslator" form="short">
+        <single>ed. &amp; trans.</single>
+        <multiple>eds. &amp; trans.</multiple>
+      </term>
+      <term name="translator" form="short">
+        <single>trans.</single>
+        <multiple>trans.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
+        <group delimiter=", ">
+          <names delimiter=", " variable="container-author">
+            <name delimiter=", " and="symbol" initialize-with=". "/>
+            <label form="short" prefix=" (" text-case="title" suffix=")"/>
+          </names>
+          <names delimiter=", " variable="editor translator">
+            <name delimiter=", " and="symbol" initialize-with=". "/>
+            <label form="short" prefix=" (" text-case="title" suffix=")"/>
+          </names>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="article-journal chapter paper-conference entry-dictionary entry-encyclopedia" match="none">
+        <group delimiter=", " prefix=" (" suffix=")">
+          <names delimiter=", " variable="container-author">
+            <name delimiter=", " and="symbol" initialize-with=". "/>
+            <label form="short" prefix=", " text-case="title"/>
+          </names>
+          <names delimiter=", " variable="editor translator">
+            <name delimiter=", " and="symbol" initialize-with=". "/>
+            <label form="short" prefix=", " text-case="title"/>
+          </names>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="symbol" initialize-with="" sort-separator=" " name-as-sort-order="all" et-al-subsequent-use-first="100" et-al-subsequent-min="101" et-al-use-first="100" et-al-min="101" delimiter-precedes-last="never"/>
```

Comment:
It's not validating because of these et-al rules here.
If you want always all authors shown, just remove those 4 et-al rules here.

Reply:
Yes I want to show all authors. Thanks for the tip!